pkg/auth/tokens/issuer_factory.go

```diff
@@ -4,8 +4,8 @@ import (
 	"encoding/json"
 	"time"
 
+	roxjwt "github.com/stackrox/rox/pkg/jwt"
```

Comment:
I think what we currently do when we have name clashes is to use `pkgX`, so in that case it'd be `pkgJWT`.